app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionTextDownloadController.scala

```diff
@@ -41,7 +41,7 @@ class EoriTextDownloadController @Inject() (
         name       <- cdsFrontendDataCache.sub02Outcome.map(_.fullName.trim)
         processedDate <- cdsFrontendDataCache.sub02Outcome
           .map(_.processedDate)
-      } yield Ok(eoriNumberTextDownloadView(eori, name, processedDate))
+      } yield Ok(subscriptionTextDownloadView(eori, name, processedDate))
```

Comment:
We can have a common controller for both standalone and combi journeys

Reply:
I have removed the separate controller and now using the common controller for both standalone and combi journeys.